classes/albums.rb

```diff
@@ -3,8 +3,12 @@
 class MusicAlbum < Item
   attr_accessor :on_spotify
 
-  def initialize(publish_date, archived, on_spotify: false, id: Random.rand(1..1000))
+  def initialize(publish_date, archived, on_spotify, id: Random.rand(1..1000))
     super(publish_date, archived, id: id)
     @on_spotify = on_spotify
   end
+
+  def can_be_archived?
+    super && !@on_spotify
```

Comment:
![image](https://user-images.githubusercontent.com/53308506/164311839-013382f8-e60e-4a7a-a235-4037d176e669.png)
Reminder of this method's requirements.

```suggestion
    super && @on_spotify
```